first-party-dependencies/pom.xml

```diff
@@ -59,10 +59,10 @@
     <gax.version>2.4.1</gax.version>
     <grpc-gcp.version>1.1.0</grpc-gcp.version>
     <guava.version>30.1.1-jre</guava.version>
-    <protobuf.version>3.18.0</protobuf.version>
+    <protobuf.version>3.17.3</protobuf.version>
     <google.api-common.version>2.0.2</google.api-common.version>
     <google.common-protos.version>2.5.0</google.common-protos.version>
-    <google.core.version>2.1.4</google.core.version>
+    <google.core.version>2.1.3</google.core.version>
```

Comment:
Google-cloud-core already depends on protobuf 3.18.0. Thus I needed to downgrade this too.